File: Python/Ejercicios condicionales Python/ejercicio3.py
# Leer un número entero y determinar si es negativo

try:
	numero = int(input("Ingrese el valor numerico: "))

	if numero < 0:
		print("El valor ingresado es negativo.")
	else:
		print("El valor ingresado no es negativo.")

except ValueError:
	print("El valor indicado no es correcto.")

File: Java/Ejercicios java print/ejercicio3.java
import java.util.Scanner;
public class main {
    public static void main(String[]args){
        int x;
        Scanner sc = new Scanner (System.in);
        System.out.println("Ingrese el valor numerico: ");
        x = sc.nextInt();
        System.out.println("El valor ingresado es: " +x);
        System.out.println("El doble es: "+x*2);
        System.out.println("El triple es: "+x*3);
    }
}